source/client/ui/story/AudioTaskView.ts

```diff
@@ -157,10 +157,19 @@ export default class AudioTaskView extends TaskView<CVAudioTask>
         }
 
         const id = element.parentElement.parentElement.id;
-        const fileProp = id == "filename" ? this.task.ins.filepath : this.task.ins.captionPath;
-        const extText = id == "filename" ? ".mp3" : ".vtt";
-
-        if(filename.toLowerCase().endsWith(extText)) {
+        const type = (id == "filename")? "audio": "subs";
+        const fileProp = (type == "audio") ? this.task.ins.filepath : this.task.ins.captionPath;
+
+        const ext = filename.toLowerCase().split(".").pop();
+        if(type === "subs" && ext != "vtt"){
+            Notification.show(`Unable to load - Only ${type === "subs"?".vtt":".mp3,.m4a"} files are currently supported.`, "warning");
```

Comment:
With this restructuring, the `type === "subs"` check isn't really necessary anymore